.github/workflows/formatting-linting.yml

```diff
@@ -35,3 +35,6 @@ jobs:
 
       - name: 'Lint subgraph ts and js'
         run: yarn lint:subgraph:ts >> $GITHUB_STEP_SUMMARY
+
```

Comment:
Should we add the ci/cd here or you want to do it as part of another PR?

Reply:
you mean for deployment to npm?

Reply:
yep

Reply:
done